test/integration/targets/win_get_url/tasks/main.yml

```diff
@@ -27,6 +30,25 @@
     dependencies:
     - tcpip
 
+- name: gathering facts
+  setup:
+
+- block:
+  # unable install python package on Win2008 due to bug in win_chocolatey module (it wants interactive input)
+  - name: install python
+    win_chocolatey:
+      name: python
+    register: installed_python
+    until: installed_python is succeeded
+
+  - name: start simple http.server
+    win_command: python -m http.server {{ http_port }}
```

Comment:
Instead of installing Python on Windows to run an HTTP server, we should probably use the httptester that is used for the other integration tests.

See the [win_uri integration tests](https://github.com/ansible/ansible/blob/devel/test/integration/targets/win_uri/) for an example of how this is currently done. You may also want to refer to the existing [get_url integration tests](https://github.com/ansible/ansible/tree/devel/test/integration/targets/get_url/).

Reply:
> Instead of installing Python on Windows to run an HTTP server, we should probably use the httptester that is used for the other integration tests.
> 
> See the [win_uri integration tests](https://github.com/ansible/ansible/blob/devel/test/integration/targets/win_uri/) for an example of how this is currently done. You may also want to refer to the existing [get_url integration tests](https://github.com/ansible/ansible/tree/devel/test/integration/targets/get_url/).

Hi, as I can see httptester is used for simple http get/put tests. It doesn't cover scenarios with custom files download. Existing [get_url integration tests](https://github.com/ansible/ansible/tree/devel/test/integration/targets/get_url/) also use SimpleHttp server running with python for that purpose. If it's possible to perform such tests with httptester - can you please share more related details?

Reply:
Sorry, I forgot we were not using the test container for those particular tasks.

While it may be possible to use the http test container with custom files, I don't think we currently have any tests that do so (modifications to the test container may be required to support file uploads).

Go ahead and continue with the current approach since that's what we're already doing for the get_url integration tests.